Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1989,7 +1989,7 @@ extension BrowserViewController: TabManagerDelegate {
             updateTabCountUsingTabManager(tabManager)
         }
         
-        if PrivateBrowsingManager.shared.isPrivateBrowsing {
+        if PrivateBrowsingManager.shared.isPrivateBrowsing && self.presentedViewController == nil {
```

Comment:
`self` isn't needed, can leave it if it's more readable to you, lmk and I will merge it